src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/MonthCalendarTests.cs

```diff
@@ -195,88 +196,74 @@ public static IEnumerable<object[]> AnnuallyBoldedDates_Set_TestData()
 
         [WinFormsTheory]
         [MemberData(nameof(AnnuallyBoldedDates_Set_TestData))]
-        public void MonthCalendar_AnnuallyBoldedDates_Set_GetReturnsExpected(DateTime[] value, DateTime[] expected)
+        public void MonthCalendar_AnnuallyBoldedDates_Set_GetReturnsExpected_IfControlNotCreated(DateTime[] value, DateTime[] expected)
         {
             using var calendar = new MonthCalendar
             {
                 AnnuallyBoldedDates = value
             };
-            Assert.Equal(expected, calendar.AnnuallyBoldedDates);
-            Assert.NotSame(value, calendar.AnnuallyBoldedDates);
-            if (value?.Length > 0)
-            {
-                Assert.NotSame(calendar.AnnuallyBoldedDates, calendar.AnnuallyBoldedDates);
-            }
-            else
-            {
-                Assert.Same(calendar.AnnuallyBoldedDates, calendar.AnnuallyBoldedDates);
-            }
-
-            Assert.False(calendar.IsHandleCreated);
 
-            // Set same.
-            calendar.AnnuallyBoldedDates = value;
             Assert.Equal(expected, calendar.AnnuallyBoldedDates);
-            Assert.NotSame(value, calendar.AnnuallyBoldedDates);
-            if (value?.Length > 0)
-            {
-                Assert.NotSame(calendar.AnnuallyBoldedDates, calendar.AnnuallyBoldedDates);
-            }
-            else
-            {
-                Assert.Same(calendar.AnnuallyBoldedDates, calendar.AnnuallyBoldedDates);
-            }
-
             Assert.False(calendar.IsHandleCreated);
         }
 
         [WinFormsTheory]
         [MemberData(nameof(AnnuallyBoldedDates_Set_TestData))]
-        public void MonthCalendar_AnnuallyBoldedDates_SetWithHandle_GetReturnsExpected(DateTime[] value, DateTime[] expected)
+        public void MonthCalendar_AnnuallyBoldedDates_Set_GetReturnsExpected_IfControlCreated(DateTime[] value, DateTime[] expected)
         {
             using var calendar = new MonthCalendar();
-            Assert.NotEqual(IntPtr.Zero, calendar.Handle);
-            int invalidatedCallCount = 0;
-            calendar.Invalidated += (sender, e) => invalidatedCallCount++;
-            int styleChangedCallCount = 0;
-            calendar.StyleChanged += (sender, e) => styleChangedCallCount++;
-            int createdCallCount = 0;
-            calendar.HandleCreated += (sender, e) => createdCallCount++;
+            calendar.CreateControl();
 
             calendar.AnnuallyBoldedDates = value;
+
             Assert.Equal(expected, calendar.AnnuallyBoldedDates);
-            Assert.NotSame(value, calendar.AnnuallyBoldedDates);
-            if (value?.Length > 0)
-            {
-                Assert.NotSame(calendar.AnnuallyBoldedDates, calendar.AnnuallyBoldedDates);
-            }
-            else
+            Assert.True(calendar.IsHandleCreated);
+        }
+
+        public static IEnumerable<object[]> AnnuallyBoldedDates_RemoveItem_TestData()
```

Comment:
If a test data is used once, name it like a test + TestData.
```suggestion
        public static IEnumerable<object[]> MonthCalendar_AnnuallyBoldedDates_RemoveItem_GetReturnsExpected_TestData()
```